src/views/InvolvementProfile/components/Membership/components/MemberList/index.js

```diff
@@ -33,7 +33,19 @@ const MemberList = ({
     return () => window.removeEventListener('resize', resize);
   });
 
+  // compare names by participation Hierarchy "Advisor" > "Leader" >
+  //"member" > "Guest" then alphabetically.
+
   const compareByLastThenFirst = (a, b) => {
```

Comment:
We should rename this function, since it no longer compares just by name. Also, there's a better way to do function comments in JavaScript which will give hints and documentation in VSCode.
```suggestion
    /**
   * Compares two members by their role and then their names - last then first
   *
   * Roles follow this hierarchy: Advisor > Leader > Member > Guest
   * @param {Object} a the first member to compare
   * @param {Object} b the second member to compare
   * @returns {-1 | 0 | 1} the sort order of the members: -1 if a before b, 1 if b before a, or 0 if equal
   */
  const compareByRoleThenLastThenFirst = (a, b) => {
```

This gives the following doc hint in VSCode:
<img width="610" alt="Screen Shot 2021-06-30 at 4 37 25 PM" src="https://user-images.githubusercontent.com/35319956/124027967-746abd80-d9c1-11eb-8084-46471b3095fc.png">

Reply:
Done!!! I really liked this way of writing comment, Thanks @EjPlatzer  for this feedback.